src/main/java/stringAddCalculator/Divider.java

```diff
@@ -0,0 +1,29 @@
+package stringAddCalculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Divider {
+    private final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private ArrayList<String> DELIMITER = new ArrayList<>(Arrays.asList(",", ":"));
```

Comment:
```suggestion
    private List<String> DELIMITER = new ArrayList<>(Arrays.asList(",", ":"));
```

구현체를 바꾸기 쉽게 List 인터페이스로 표시해주는걸 추천드립니다.